content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -420,6 +420,13 @@ datasource db {
 
 ##### Analyze the database and write its schema to the `schema.prisma` file
 
+<Admonition type="warning">
+
+**Deprecation warning** <br />
+From Prisma 3.0.0 onwards, the `prisma introspect` command will be deprecated and replaced with the [`prisma db pull`](#db-pull) command.
+
+</Admonition>
+
```

Comment:
Could make sense to move the warning and have it next to "**MongoDB not supported**" I think